src/write.jl

```diff
@@ -106,14 +106,18 @@ function write_las(io::IO, las::LasDataset, compressed::Bool = false)
         user_fields = ()
         # find indices of existing extra bytes VLRs
         extra_bytes_idxs = findall(vlr -> (get_user_id(vlr) == LAS_SPEC_USER_ID) && (get_record_id(vlr) == ID_EXTRABYTES), vlrs)
-        # need to adjust the data record length in the header to remove these extra bytes
-        for i ∈ extra_bytes_idxs
-            header.data_record_length -= sizeof(data_type(get_data(vlrs[i])))
+        if !isempty(extra_bytes_idxs)
+            @assert length(extra_bytes_idxs) == 1 "Found $(length(extra_bytes_idxs)) Extra Bytes VLRs when we should only have 1"
```

Comment:
with this asserting check here, there will be just one extra_bytes_idxs after this line. why you are deleting that in lines 117-119?

Reply:
It was to grab the index and delete the match we found, but we don't need this code anymore :)